api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,62 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
+        description: |
+          An array of subject attributes that are required for updating the event type. Any one of the attributes
+          defined in this array is sufficient to be authorized. The wildcard item takes precedence over all others,
+          i.e. if it is present, all users are authorized.
+        items:
+          $ref: '#/definitions/AuthorizationAttribute'
+        minItems: 1
+      read:
+        type: array
+        description: |
+          An array of subject attributes that are required for reading events from the event type. Any one of the
+          attributes defined in this array is sufficient to be authorized. The wildcard item takes precedence over
+          all others, i.e., if it is present, all users are authorized.
+        items:
+          $ref: '#/definitions/AuthorizationAttribute'
+        minItems: 1
+      write:
+        type: array
+        description: |
+          An array of subject attributes that are required for writing events to the event type. Any one of the
+          attributes defined in this array is sufficient to be authorized. The wildcard item takes precedence over
+          all others, i.e. if it is present, all users are authorized.
+        items:
+          $ref: '#/definitions/AuthorizationAttribute'
+        minItems: 1
+    required:
+      - admin
+      - read
+      - write
+
+  AuthorizationAttribute:
+    type: object
+    description: |
+      An attribute for authorization. This object includes the type of attribute (team, permission, etc.),
+      and the attribute value. A wildcard can be represented with type '*', and value '*'. It means that all
+      authenticated users are allowed to perform an operation.
+    properties:
+      attribute_type:
+        type: string
+        description: the type of attribute (e.g., 'team', or 'permission')
```

Comment:
I'm a little worried about that, because possible values will depend on the particular deployment. The authorization module is actually in a separate repository, as it is Zalando-specific, so it is service-dependent, in my opinion. We are however looking into exposing various configuration values, including a list of accepted attribute types, either as a new endpoint, or in our GHE wiki.